app/src/main/res/layout/faq_activity.xml

```diff
@@ -1,7 +1,36 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:tools="http://schemas.android.com/tools"
-  android:id="@+id/faq_fragment_placeholder"
-  android:layout_width="match_parent"
-  android:layout_height="match_parent"
-  tools:context=".help.faq.FAQActivity" />
+<layout>
+
+  <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    tools:context=".help.faq.FAQActivity">
+
+    <com.google.android.material.appbar.AppBarLayout
+      android:id="@+id/faq_app_bar_layout"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:theme="@style/ThemeOverlay.AppCompat.Dark.ActionBar">
+
+      <androidx.appcompat.widget.Toolbar
+        android:id="@+id/faq_activity_toolbar"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="end"
+        android:background="@color/colorPrimary"
+        android:minHeight="?attr/actionBarSize"
+        android:theme="@style/Widget.AppCompat.ActionBar"
+        app:titleTextColor="@color/white" />
+    </com.google.android.material.appbar.AppBarLayout>
+
+    <FrameLayout
+      android:id="@+id/faq_fragment_placeholder"
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      tools:context=".help.faq.FAQActivity" />
```

Comment:
Remove this line. This is not needed here.

Reply:
Done :)